README.md

```diff
@@ -1 +1,66 @@
-# Math-Magicians
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+- Make simple calculations.
+- Read a random math-related quote.
+
+## Built With
+
+- React ⚛️
+
+<!-- ## Live Demo
+
+[Live Demo](https://rmrmrmr.github.io/Math-Magicians) -->
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a source code editor such as VSCode
```

Comment:
- [ ] Please stick to [Microverse README template](https://github.com/microverseinc/readme-template) and add the table of contents with links to each section title.